source/core/parameters_lagrangian.cc

```diff
@@ -466,6 +466,14 @@ namespace Parameters
                           "Choosing particle-wall contact force model"
                           "Choices are <pw_linear|pw_nonlinear>.");
 
+        prm.declare_entry(
+          "rolling resistance torque method",
+          "no_rolling_resistance_torque",
+          Patterns::Selection(
+            "no_rolling_resistance_torque|constant_rolling_resistance_torque|viscous_rolling_resistance_torque"),
```

Comment:
Wouldnt it be better to just call them no_resistance, constant_resistance and viscous_resistance? it would be as easy to understand but less verbose

Reply:
Changed.